### PR TITLE
Greatly reduce subtest set to basic workflow

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: config_custom/control.ini [hunk 1]
```diff
@@ -1,23 +1,9 @@
...
-          ,docker_cli/events
-          ,docker_cli/images_all
-          ,docker_cli/liverestore
-          ,docker_cli/load
-          ,docker_cli/login
-          ,docker_cli/pull
-          ,docker_cli/run_attach
-          ,docker_cli/run_cgroups
-          ,docker_cli/run_exec/exec_false
-          ,docker_cli/run_exec/exec_pid_count
```
Comment (edsantiago) [inline config_custom/control.ini]: exec_pid_count has just been fixed, next build should include it. All the rest of run_exec passes; I think run_exec should be included.

File: config_custom/control.ini [hunk 2]
```diff
@@ -1,23 +1,9 @@
...
-          ,docker_cli/run_sigproxy
-          ,docker_cli/selinux_labels
-          ,docker_cli/stop/stopped
-          ,docker_cli/systemd
```
Comment (edsantiago) [inline config_custom/control.ini]: systemd actually works. Why exclude it?

File: config_custom/control.ini [hunk 3]
```diff
@@ -1,23 +1,9 @@
...
-          ,redhat/rhel_push_plugin
+subtests = docker_cli/create,
+           docker_cli/dockerhelp,
+           docker_cli/import_export,
+           docker_cli/run,
+           docker_cli/start,
+           docker_cli/stop,
+           docker_cli/version,
```
Comment (edsantiago) [inline config_custom/control.ini]: The version test is useless for podman; I see no reason to include it.

Comment (edsantiago) [reply config_custom/control.ini]: Some that I think worth including: attach, commit, run_cgroup_parent, build, kill_stopped, logs, wait, run_cidfile, kill, kill_bad, rm, history

**Comment by cevich** [comment]
@edsantiago okay, all suggestions added.  I'm not sure about ``build``, will have to do a test-run and see...

**Comment by edsantiago** [comment]
Did I say build? (github lost my comment; or at least shoved it somehwre I can't find it). No, build is still failing with no fix in sight. Leave it out. Sorry for including that, I didn't mean to.

**Comment by cevich** [comment]
> Did I say build?

Yep, hehe, okay, no worries, I'll yank it  :smile: 

> Some that I think worth including: attach, commit, run_cgroup_parent, **build**, kill_stopped, logs, wait, run_cidfile, kill, kill_bad, rm, history

**Comment by cevich** [comment]
@edsantiago I'm getting failures in:

* login
* pull
* run_exec
* systemd

I didn't look deeply, but [they're probably related to the configuration (as per usual)](https://github.com/autotest/autotest-docker/blob/podman/config_custom/subtests.ini).

I'm going to leave them out for now, since this really is just a smoke-test job.  Reliably passing is more important than coverage.  Adding a more complete job is a task for another day.

**Comment by edsantiago** [comment]
This must be RHEL; I've never played with podman on RHEL. Do you have a link to the results?

**Comment by cevich** [comment]
> This must be RHEL

Well...let's just merge this and we'll have them tomorrow :grinning:

**Pull request merged**